modules/apps/commerce/commerce-currency-service/src/main/java/com/liferay/commerce/currency/internal/upgrade/registry/CommerceCurrencyServiceUpgradeStepRegistrator.java

```diff
@@ -57,6 +64,12 @@ protected String[] getModuleTableNames() {
 
 			});
 
+		registry.registerInitialDeploymentUpgradeSteps(
```

Comment:
@JanisZhang,

You shouldn't register registerInitialDeploymentUpgradeSteps in a regular upgrade registrator class. You should create a specific class, like in CommerceAccountServiceInitialDeploymentUpgradeStepRegistrator.